friends/src/App.js

```diff
@@ -0,0 +1,94 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import './App.css';
+import FriendsList from './components/FriendsList';
+import Friend from './components/Friend';
+import FriendForm from './components/FriendForm';
+import EditForm from './components/EditForm';
+import {Route } from 'react-router-dom';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(res => {
+        //console.log(res);
+        this.setState({friends: res.data});
+      })
+      .catch(err => console.log('err', err));
+  }
+
+  addFriend = newFriend => {
+    // add new friend if all fields are filled out
+    if (newFriend.name && newFriend.age && newFriend.email) {
+      axios
+        .post('http://localhost:5000/friends', newFriend)
+        .then(res => {
+          this.setState({friends: res.data});
+        })
+        .catch(err => console.log(err));
```

Comment:
Why have an extra step here in your callback? You can just use `console.log` since `console` is an object and `log` a method on it that'll take however many arguments you pass it and log them:
```javascript
.catch(console.log); /* Error object is passed to console.log implicitly, 
just like an Event object being passed implicitly in event handlers for onClicks */
```